contributions/catalog/templates/auth/500_error.html

```diff
@@ -0,0 +1,7 @@
+{% extends "base.html" %}
```

Comment:
I was not able to see 500 error when I tried to submit without mongodb. The submit page just went to the new submit page after clicking the submit button instead of giving an error of db connection in front end. The back end gave an error.